CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.14.0
 
+- Adds new 'limiter' class for setting max page width across screen sizes.
+- Significantly refactors layout scale to be more consistent across different properties and to increment more predictibly.
```

Comment:
I always find it really helpful for the changelog to point out the specific changes ("added classes x, y, z", "removed classes x, y, z"), so I can use the changelog entry as an "upgrade guide" — telling me exactly what I need to look for and alter in my code before upgrading.